ios/engine/KMEI/KeymanEngineTests/TestUtils/Downloading/URLSessionMock.swift

```diff
@@ -38,14 +39,14 @@ extension TestUtils.Downloading {
       case mockQueueEmpty
     }
 
-    private var mockedResultQueue: [Task]
+    private var mockedResultQueue: [(Task, XCTestExpectation?)]
```

Comment:
New test feature:  mocked URL tasks may now accept a `XCTestExpectation`, allowing us to ensure they occur as part of a test.

This helps support one of the new package-installer cancellation tests.  It's possible to work around that by adding more mocking structure (as I did for some older tests), but this looks much cleaner and more easily manageable.